test/gnmi/subscribetest.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019-present Open Networking Foundation.
+// Copyright 2020-present Open Networking Foundation.
```

Comment:
probably shouldn't update if the file existed in 2019

Reply:
Good catch. Not sure why I did that...